docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -3,18 +3,19 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= OpenID Connect Client and Token Propagation Quickstart
+= OpenID Connect client and token propagation quickstart
 include::_attributes.adoc[]
 :categories: security
-:summary: This guide explains how to use OpenID Connect and OAuth2 Client and Filters to acquire, refresh and propagate access tokens.
 :topics: security,oidc,client
 :extensions: io.quarkus:quarkus-oidc-client
 
-This quickstart demonstrates how to use `OpenID Connect Client Reactive Filter` to acquire and propagate access tokens as `HTTP Authorization Bearer` access tokens, alongside `OpenID Token Propagation Reactive Filter` which propagates the incoming `HTTP Authorization Bearer` access tokens.
+You can use OpenID Connect (OIDC) and OAuth2 client and filters to acquire, refresh, and propagate access tokens.
 
-Please check xref:security-openid-connect-client-reference.adoc[OpenID Connect Client and Token Propagation Reference Guide] for all the information related to `Oidc Client` and `Token Propagation` support in Quarkus.
+This approach uses an OIDC token propagation Reactive filter to propagate the incoming HTTP authorization bearer access tokens.
```

Comment:
```suggestion
This approach uses an OIDC token propagation Reactive filter to propagate the incoming bearer access tokens.
```

 `HTTP Authorization` is a header so probably better to drop `HTTP authorization`

Reply:
Updated. Thanks.

Reply:
Hi @rolfedh - still seeing `HTTP authorization` - can you please push the minor update ?

Reply:
Sorry. It took me a while to push the changes. When I made that update, I realized that two of the files still needed significant work, which I just finished.